FinniversKit/Sources/Recycling/GridViews/NeighborhoodProfile/NeighborhoodProfileView.swift

```diff
@@ -20,7 +20,30 @@ public final class NeighborhoodProfileView: UIView {
 
     // MARK: - Private properties
 
-    private var viewModel = NeighborhoodProfileViewModel(title: "", readMoreLink: nil, cards: [])
+    private var viewModel = NeighborhoodProfileViewModel(title: "", readMoreLink: nil, cards: [], banner: nil)
+
+    private lazy var bannerView: NeighborhoodProfileBannerView = {
+        let view = NeighborhoodProfileBannerView(withAutoLayout: true)
+        view.delegate = self
+        return view
+    }()
+
+    private func centeredContainerView(with views: [UIView]) -> UIStackView {
+        let stack = UIStackView(withAutoLayout: true)
+        stack.axis = .horizontal
```

Comment:
Same here regarding the init.

```suggestion
        let stack = UIStackView(axis: .horizontal, withAutoLayout: true)
```